packages/web-frontend/src/selectors/measureSelectors.js

```diff
@@ -248,7 +248,7 @@ export const selectAreMeasuresOnTheSameEntityLevel = createSelector(
     }
 
     const measureLevelsFromMeasures = Object.values(measureInfo)
-      .map(({ measureLevel }) => measureLevel)
+      .map(({ measureLevel = [] }) => measureLevel)
```

Comment:
I cannot find a way to reproduce the error, but here is the only place to read `measureLevel` from an object.